OrbitGl/App.cpp

```diff
@@ -17,14 +17,20 @@
 #include "Callstack.h"
 #include "Capture.h"
 #include "CaptureSerializer.h"
+#ifndef NOGL
```

Comment:
The double negative here and below is a little harder to parse than something like #ifdef GL. Would it make sense to change it or does that cause other weirdnesses elsewhere? I understand you want default behavior when nothing is defined explicitly, so probably better to leave as-is.

Reply:
Yes, that was the intention. Not to change the default behaviour. So I would prefer to leave it like so.

Reply:
Ok, sounds good!